packages/helpers/source/censor.ts

```diff
@@ -738,6 +738,6 @@ export class Censor {
 	 * @memberof Censor
 	 */
 	#removeSpam(value: string): string {
-		return new Censorify().process(value);
+		return new Censorify.default().process(value);
```

Comment:
Need to check here if I really have to reference it like that or if there's a nicer way.